pyccel/ast/core.py

```diff
@@ -2783,7 +2782,7 @@ def __init__(self, func, args, current_function=None):
         if str(current_function) == str(name):
             func.set_recursive()
 
-        if not isinstance(args, (tuple, list, Tuple)):
+        if not isinstance(args, (tuple, list)):
             raise TypeError('> expecting an iterable')
```

Comment:
Could you please change this error message?
A general iterable object is not accepted here, it must be either `list` or `tuple`.